Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1095,7 +1123,7 @@ OcIsSandyOrIvy (
 
   DEBUG ((
     DEBUG_VERBOSE,
-    "OCCPU: Discovered CpuFamily %d CpuModel %d SandyOrIvy %d\n",
+    "OCCPU: Discovered CpuFamily %d CpuModel %d SandyOrIvy %a\n",
```

Comment:
SandyOrIvy is boolean, thus decimal, and not an ASCII string.

Reply:
Doh! I had started going through and fixing up all the format strings and rolled some of them back but forgot this one. I was going to change it to an ASCII string and then pass either `SandyOrIvy ? "YES" : "NO"`. This is one of those places I mentioned in a previous comment where the code is passing an `unsigned char` (`BOOLEAN`) to the formatter `%d`. I'm not sure that's okay as it certainly wouldn't be okay if this were `printf`.

Reply:
Mhm, why is that not ok with C printf? Will not integer promotion promote it to int, thus %d? I think we use %d in many places for BOOLEn, and it makes sense to me to continue to to be honest.

Reply:
Huh, today I learned. I didn't realize types smaller than `int` were promoted to `int` in variadic function arguments and functions without prototypes. Ignore everything I was saying about the use of `UINT16` and `BOOLEAN` with `%d`. I'm an idiot.